third_party/mesa/src/freedreno/common/freedreno_devices.py

```diff
@@ -204,7 +204,7 @@ def __str__(self):
         gmem_align_w = 32,  gmem_align_h = 32,
         tile_align_w = 32,  tile_align_h = 32,
         tile_max_w   = 512,
-        tile_max_h   = ~0, # TODO
+        tile_max_h   = 0, #GOOGLE, change this to 0 to fix a warning in generated header, original value: ~0, # TODO
```

Comment:
i am using the generated header in Dive to check the gpu id (will submit in a different PR), and because of the warning, the compiling failed.

Reply:
in the generated header, tile_max_h has a type of uint32_t, and ~0 will generate -1, so the warning is implicit converting from int to uint32_t. without my change, it also has #TODO (so i assume it is a temp one), and i assume eventually someone from mesa side will update the value to a correct one.

Reply:
Setting the max tile height to 0 doesn't cause any issues?

Reply:
no, it is not used for now. `~0` is already a broken value

Reply:
Wait, doesn't setting a -1 to a uint32_t essentially sets it to UINT32_MAX? In that case, that means setting an unbounded max tile height?

Reply:
change the value to 4294967295 which is UINT32_MAX, so should be safe now :)

Reply:
Nitpicky: Is it possible to set it to 0xffffffff instead or  2^32–1?
Seems less error prone in case you get one of those digits wrong.
Nitpicking because eventually we'll upstream this change, so want to make sure we do the cleanest job possible.

Reply:
updated the file as suggested. :)